### DISCUSSION TITLE
How to escape `space` in kitten path?

### DISCUSSION THREAD

**Discussion by mcchrish** [mikesmithgh/kitty-scrollback.nvim#293] [Q&A] [answered]
It's more of a general question about kitty config rather than the plugin.

I have a config line in kitty like this:

```
action_alias kitty_scrollback_nvim kitten '~/Library/Application Support/nvim/lazy/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py'
```

But it doesn't work because the the space between `Application Support`.

Also tried escape:

```
action_alias kitty_scrollback_nvim kitten ~/Library/Application\ Support/nvim/lazy/kitty-scrollback.nvim/python/kitty_scrollback_nvim.py
```

**Comment by mikesmithgh** [comment, marked as answer]
I took a quick look, and I don't believe you can escape a space in an argument to `action_alias` at the moment. I'll look into this a bit more to confirm.

You can workaround this by using a symlink to your `~/.config/kitty` directory. For example,

```sh
ln -s '~/Library/Application Support/nvim/lazy/kitty-scrollback.nvim' ~/.config/kitty/kitty-scrollback.nvim
```

Then in your kitty.conf, do:

```
action_alias kitty_scrollback_nvim kitten kitty-scrollback.nvim/python/kitty_scrollback_nvim.py
```

@mcchrish Let me know if that works, thanks!

**Reply by mcchrish** [reply]
Thanks @mikesmithgh! Symlink workaround works and `.config/kitty` is a perfect place since kitty uses anyway.

**Reply by mikesmithgh** [reply]
Awesome. I opened an issue https://github.com/kovidgoyal/kitty/issues/8178 against the Kitty repo for this 👍

**Reply by mikesmithgh** [reply]
@mcchrish https://github.com/kovidgoyal/kitty/issues/8178 is fixed on Kitty nightly now 🥳 . So, when the next version of Kitty is released you can just wrap the path in quotes and will no longer need the symlink. I've also added some comments during the `:KittyScrollbackGenerateKittens` command to let the user know if the plugin is in a path that contains whitespace.
